From c69ebec307323af692b51fa6047cd8c4803a7063 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Mon, 10 Feb 2025 09:31:58 +0100
Subject: [PATCH 1604/1695] dts: stm32: add wakeup properties to the RTC for
 stm32mp21

The 'wakeup-source' property allows to say that the RTC peripheral
can wakeup the platform.
Overload the property 'wakeup-parent' for the RTC as the wakeup-parent
capable of wakeup the platform with a RTC signal input is the exti2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 0da3ee47c1dbc9106a8ddeb804cff7cf27c78dce dts: stm32: add wakeup properties to the RTC for stm32mp21]
Change-Id: Iacc356f1c3eab8fcc82d96e88fc49979bb686322
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/435774
---
 core/arch/arm/dts/stm32mp211.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 7a7d3f6ab..f956ae7ef 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -1081,6 +1081,7 @@
 			reg = <0x46000000 0x400>;
 			clocks = <&rcc CK_BUS_RTC>, <&rcc RTC_CK>;
 			clock-names = "pclk", "rtc_ck";
+			wakeup-source;
 			interrupts-extended = <&exti2 22 IRQ_TYPE_EDGE_RISING>;
 		};
 
-- 
2.47.2

